### ISSUE TITLE
TS `satisfies` operator can't be parsed

### ISSUE THREAD

**Issue by Andarist** [benjamn/recast#1281]
Parsing a file like:
```ts
export default {} satisfies {}
```
end up crashing with:
```
Error: did not recognize object of type "TSSatisfiesExpression"
```

**Comment by benjamn** [comment]
@Andarist Can you try `npm i recast@next` to get version 0.23.1?

[Support for `TSSatisfiesExpression`](https://github.com/benjamn/recast/blob/5e6442d2d66bb00af581b8d08b2ac970bd6d4a5c/lib/printer.ts#L2289-L2298) should have been implemented in #1232 (https://github.com/benjamn/recast/commit/e51e1bfe490a8d7784ba8b387366cd8e7388d540).

If that works, I will promote `recast@0.23.1` to `latest`.

**Comment by Andarist** [comment]
I can confirm that this issue is resolved in `0.23.1`
